src/java/com/sarality/app/view/EditorView.java

```diff
@@ -0,0 +1,18 @@
+package com.sarality.app.view;
+
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputConnection;
+
+/**
+ * Interface for a View that has editable elements
```

Comment:
If you look at the Output Composer it needs access to the Input Connection and the EditorOptions to be able to add things to the editable portions of the View. 

Of course this is not needed when you use the Flight Assistant Input Method which has access to the InputCOnnection as well as Editor Options but if you want to use things without requiring the user to change the Keyboard, you need some mechanism to incept the creation of the InputCOnnection. This interface provides that mechanism